editor/plugins/sprite_frames_editor_plugin.cpp

```diff
@@ -31,10 +31,25 @@
 #include "sprite_frames_editor_plugin.h"
 
 #include "core/io/resource_loader.h"
+#include "core/os/keyboard.h"
 #include "core/project_settings.h"
 #include "editor/editor_settings.h"
 #include "scene/3d/sprite_3d.h"
 
+void SpriteFramesEditor::_process_input(const Ref<InputEvent> &p_event, Node *p_from) {
+	Ref<InputEventKey> k = p_event;
+
+	if (k.is_valid() && k->is_pressed() && k->get_scancode() == KEY_DELETE && !k->is_echo()) {
+		if (p_from->get_class() == "Tree") {
```

Comment:
You can probably check for `p_from == tree` directly.

Reply:
Fixed, thanks :)